packaging/systemd/configure-ovs.sh

```diff
@@ -757,9 +746,6 @@ if [ "$1" == "OVNKubernetes" ]; then
     fi
   done < <(nmcli -g NAME c)
   connections+=(ovs-if-phys0 ovs-if-br-ex)
-  if [ -f "$extra_bridge_file" ]; then
```

Comment:
the $extra_bridge_file is used in other places in the script, do we want to remove them all at once in this PR or leave it until the detach ovnk gateway change lands in microshift?

Reply:
We will remove most of the content from configure-ovs.sh in 4.14 when the detach ovnk gateway change lands. All the NM configurations will be removed when users upgrade to 4.14.

When we backport this PR to 4.13, for clusters that have br-ex1 configured, we need to keep `extra_bridge_file` in the script to ensure the behavior of `get_default_interface` is unchanged.

Reply:
ack+.